openpype/tools/project_manager/project_manager/widgets.py

```diff
@@ -264,9 +299,35 @@ def _on_ok_clicked(self):
 
         project_name = self.project_name_input.text()
         project_code = self.project_code_input.text()
-        library_project = self.library_project_input.isChecked()
-        create_project(project_name, project_code, library_project)
+        project_width = self.project_width_input.text()
+        project_height = self.project_height_input.text()
+        project_fps = self.project_fps_input.text()
+        project_aspect = self.project_aspect_input.text()
+        project_frame_start = self.project_frame_start_input.text()
+        project_frame_end = self.project_frame_end_input.text()
```

Comment:
Use integer or float type instead of string.
```suggestion
        project_width = self.project_width_input.value()
        project_height = self.project_height_input.value()
        project_fps = self.project_fps_input.value()
        project_aspect = self.project_aspect_input.value()
        project_frame_start = self.project_frame_start_input.value()
        project_frame_end = self.project_frame_end_input.value()
```

Reply:
thanks!